library/core/lifecycle-events/src/main/java/org/quiltmc/qsl/lifecycle/api/event/ServerWorldTickEvents.java

```diff
@@ -0,0 +1,64 @@
+package org.quiltmc.qsl.lifecycle.api.event;
+
+import org.quiltmc.qsl.base.api.event.ArrayEvent;
+
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.server.world.ServerWorld;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+
+@Environment(EnvType.CLIENT)
```

Comment:
Why is this common class only present on the client??

Reply:
LOL dumb mistake